src/common/utxobased/db/NewModels/baselet.ts

```diff
@@ -0,0 +1,86 @@
+import { BaseType } from 'baselet'
```

Comment:
A lot of this code is dup. Seems like there's less merit to renaming the old file (`baseletsOld.ts`) and rather we could just label certain declarations here with a `// deprecated` and `// deprecated (use LastUsedByFormatPath)` comments. this way we can clearly see which parts of the API is new and which parts will be removed/replaced.

Reply:
Yes, this looks cleaner to me as well. I chose this approach in the second commit of the pull request now: https://github.com/EdgeApp/edge-currency-plugins/pull/167/commits/8921fdd949d4f7718b89a151e7b63248911b1e2c